HACKING.rst

```diff
@@ -413,10 +413,11 @@ In more detail:
 
 After the initial structure is in place, the work in this refactor will
 consist of replacing the ``cloudinit.net.some_func`` call in each
-``cloudinit.distros.Networking`` method with the actual implementation.
-This can be done incrementally, one function at a time:
+``cloudinit.distros.networking.Networking`` method with the actual
+implementation.  This can be done incrementally, one function at a
+time:
 
-* pick an unmigrated ``cloudinit.distros.Networking`` method
+* pick an unmigrated ``cloudinit.distros.networking.Networking`` method
 * refactor all of its callers to call the ``distro.net`` method on
```

Comment:
distro.networking

Reply:
Similarly, let me rework things to make it a bit clearer.